json/jsonparse.cc

```diff
@@ -406,8 +406,11 @@ void json_import_ports(Context *ctx, const string &modname, const std::vector<Id
             wire_node = wire_group_node->data_array[index];
             //
             // Pick a name for this port
+            int ndx = index + start_offset;
+            if (!upto)
```

Comment:
This looks like the wrong way round to me - when upto isn't set, we surely want to keep the old behaviour?

Reply:
Agree, but in that case yosys should always set "upto" to 1 or 0, right now we can not distinguish between 0 and old behavior. cc: @cliffordwolf

Reply:
We should certainly not be silently breaking JSON files from older Yosys